tests/spec/path-resolution.spec.js

```diff
@@ -19,7 +20,7 @@ describe('path resolution', function() {
         if(error) throw error;
 
         expect(result['node']).to.exist;
-        var _node = result['node'];
+        let _node = result['node'];
```

Comment:
Same thing here.  In fact, I don't think I see any `let` below that can't become a `const`.  Do you want to try changing them all, and see what eslint and the tests say?